sessionctx/variable/sysvar.go

```diff
@@ -1779,13 +1779,13 @@ var defaultSysVars = []*SysVar{
 		return nil
 	}},
 	// variable for top SQL feature.
-	{Scope: ScopeGlobal, Name: TiDBEnableTopSQL, Value: BoolToOnOff(DefTiDBTopSQLEnable), Type: TypeBool, Hidden: true, AllowEmpty: true, GetGlobal: func(s *SessionVars) (string, error) {
+	{Scope: ScopeGlobal, Name: TiDBEnableTopSQL, Value: BoolToOnOff(DefTiDBTopSQLEnable), Type: TypeBool, GetGlobal: func(s *SessionVars) (string, error) {
```

Comment:
'Top SQL' is still experimental, do you have the plan to introduce `tidb_experimental_feature_switch` later?

Reply:
In [the proposal](https://github.com/pingcap/tidb/blob/master/docs/design/2021-07-29-hidden-sysvars.md) it is unstable, but my understanding is that it is now stable. See: https://docs.pingcap.com/tidb/stable/basic-features

If this is incorrect, we'll need to change both locations.